plugins/node/opentelemetry-instrumentation-mysql/src/instrumentation.ts

```diff
@@ -343,4 +366,43 @@ export class MySQLInstrumentation extends InstrumentationBase<
       };
     };
   }
+  private _setPoolcallbacks(
+    pool: mysqlTypes.Pool | mysqlTypes.PoolCluster,
+    thisPlugin: MySQLInstrumentation
+  ) {
+    pool.on('connection', connection => {
+      thisPlugin._connectionsUsage.add(1, {
+        'db.client.connection.usage.state': 'idle',
```

Comment:
is `'db.client.connection.usage.{state,name}'` in semantic conventions?

Reply:
I see state but not name https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/database-metrics.md#connection-pools

Reply:
You're right. I'll remove `name`